packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -1574,10 +1574,24 @@ export class Domain extends DomainBase implements IDomain, ec2.IConnectable {
       }
     }
 
+    const unSupportEbsInstanceType = [
+      ec2.InstanceClass.I3,
+      ec2.InstanceClass.R6GD,
+      ec2.InstanceClass.I4G,
+      ec2.InstanceClass.I4I,
+      ec2.InstanceClass.IM4GN,
+      ec2.InstanceClass.R7GD,
+    ];
+
+    const supportInstanceStorageInstanceType = [
+      ec2.InstanceClass.R3,
```

Comment:
I'm not sure whether R3 should be included in `unSupportEbsInstanceType` because I couldn't find any documentation, and it has already been deprecated.

To avoid breaking changes, I decided to separate `unSupportEbsInstanceType` and `supportInstanceStorageInstanceType` in line with the current implementation, just in case.

Reply:
You should still be able to deploy previous gen instances to test this, but it probably doesn't matter either way